invenio_communities/assets/semantic-ui/js/invenio_communities/search/components.js

```diff
@@ -0,0 +1,409 @@
+          value={keyField}
+          onClick={() => onFilterClicked(keyField)}
+          checked={isSelected}
+        />
+      </div>
+      <div className={`content facet-content ${isActive ? "active" : ""}`}>
+        {childAggCmps}
+      </div>
+    </List.Item>
+  );
+};
+
+export const SearchHelpLinks = () => {
+  return (
+    <Overridable id={"RdmSearch.SearchHelpLinks"}>
+      <Grid className="padded-small">
+        <Grid.Row className="no-padded">
+          <Grid.Column></Grid.Column>
+        </Grid.Row>
+        <Grid.Row className="no-padded">
+          <Grid.Column>
+            <Card className="borderless-facet">
+              <Card.Content>
+                <a href="/help/search">"Search guide"</a>
+              </Card.Content>
+            </Card>
+          </Grid.Column>
+        </Grid.Row>
+      </Grid>
+    </Overridable>
+  );
+};
+
+export const RDMRecordFacets = ({ aggs, currentResultsState }) => {
+  return (
+    <aside aria-label="filters" id="search-filters">
+      <Toggle
+        title="Versions"
+        label="View all versions"
+        filterValue={["allversions", "true"]}
+      />
+      {aggs.map((agg) => {
+        return (
+          <div className="ui accordion" key={agg.title}>
+            <BucketAggregation title={agg.title} agg={agg} />
+          </div>
+        );
+      })}
+      <SearchHelpLinks />
+    </aside>
+  );
+};
+
+export const RDMBucketAggregationElement = ({ title, containerCmp }) => {
+  return (
+    <Card className="borderless-facet">
+      <Card.Content>
+        <Card.Header>{title}</Card.Header>
+      </Card.Content>
+      <Card.Content>{containerCmp}</Card.Content>
+    </Card>
+  );
+};
+
+export const RDMToggleComponent = ({
+  updateQueryFilters,
+  userSelectionFilters,
+  filterValue,
+  label,
+  title,
+  isChecked,
+}) => {
+  const _isChecked = (userSelectionFilters) => {
+    const isFilterActive =
+      userSelectionFilters.filter((filter) => filter[0] === filterValue[0])
+        .length > 0;
+    return isFilterActive;
+  };
+
+  const onToggleClicked = () => {
+    updateQueryFilters(filterValue);
+  };
+
+  var isChecked = _isChecked(userSelectionFilters);
+  return (
+    <Card className="borderless-facet">
+      <Card.Content>
+        <Card.Header>{title}</Card.Header>
+      </Card.Content>
+      <Card.Content>
+        <Checkbox
+          toggle
+          label={label}
+          name="versions-toggle"
+          id="versions-toggle"
+          onClick={onToggleClicked}
+          checked={isChecked}
+        />
+      </Card.Content>
+    </Card>
+  );
+};
+
+export const RDMCountComponent = ({ totalResults }) => {
+  return <Label>{totalResults.toLocaleString("en-US")}</Label>;
+};
+
+export const RDMEmptyResults = (props) => {
+  const queryString = props.queryString;
+  return (
+    <>
+      <Segment placeholder textAlign="center">
+        <Header icon>
+          <Icon name="search" />
+          "No results found!"
```

Comment:
translation string missing, also in other places

Reply:
It will be addressed in https://github.com/inveniosoftware/invenio-communities/issues/433.